src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/AESetup.java

```diff
@@ -153,25 +153,36 @@
      */
     @BeforeAll
     public static void setUpConnection() throws TestAbortedException, Exception {
-        AETestConnectionString = connectionString + ";sendTimeAsDateTime=false" + ";columnEncryptionSetting=enabled";
-
         if (null == applicationClientID || null == applicationKey || null == keyIDs
                 || (isWindows && null == windowsKeyPath)) {
             fail(TestResource.getResource("R_reqExternalSetup"));
-
         }
 
         readFromFile(Constants.JAVA_KEY_STORE_FILENAME, "Alias name");
 
-        String enclaveAttestationUrl = getConfiguredProperty("enclaveAttestationUrl");
-        if (null != enclaveAttestationUrl) {
-            AETestConnectionString = TestUtils.addOrOverrideProperty(AETestConnectionString, "enclaveAttestationUrl",
-                    enclaveAttestationUrl);
-        }
-        String enclaveAttestationProtocol = getConfiguredProperty("enclaveAttestationProtocol");
-        if (null != enclaveAttestationProtocol) {
-            AETestConnectionString = TestUtils.addOrOverrideProperty(AETestConnectionString,
-                    "enclaveAttestationProtocol", enclaveAttestationProtocol);
+        // CI unix tests with localhost only
+        String[] identifiers = connectionString.substring(Constants.JDBC_PREFIX.length()).split(Constants.SEMI_COLON);
+        if (!identifiers[0].contains("localhost")) {
+            AETestConnectionString = new String[enclaveServer.length];
```

Comment:
If the user doesn't specify enclaveServer, AETestConnectionString is not set to anything? Have you tested this case?